docs/getting-started.md

```diff
@@ -0,0 +1,102 @@
+---
+date: '2019-10-22'
+title: 'Virto Commerce Getting started'
+layout: docs
+---
+
+
+# Getting started
+
+If you are new to Virto Commerce, follow these steps to set up and launch your store.
+
+## Step 1. Create a catalog
+
+The catalog is what your customers are constantly working with. It is very important that the catalog is easy to browse and find the right products.
+
+Typically, to start working with the new catalog you should follow several steps:
+
+* Create a new catalogue
+* Create categories
+* Create a product
+* Create a product with variants
+
+Additionally you can read [Working with Products catalog](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/merchandise-management/products-catalog.md) article. It describes how to work with products [catalog](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/merchandise-management/products-catalog.md#common-catalogs), [categories](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/merchandise-management/products-catalog.md#categories), [items](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/merchandise-management/products-catalog.md#items-products), product [properties](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/merchandise-management/products-catalog.md#properties)), product [variations](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/merchandise-management/products-catalog.md#variations) and [virtual catalogs](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/merchandise-management/products-catalog.md#virtual-catalogs)
+
+## Step 2. Configure your store
+
+Virto Commerce platform is 4-MULTI: multi-language, multi-currency, multi-theme and multi-store by design. This gives the ability to have and run multiple stores on the same system.
+
+To configure your store you must accomplish a set of actions:
+
+1. Create first store
+2. Select your catalog
+3. Select available currency, languages
+4. Configure advanced properties:
+    1. Store description, emails and URLs
+    2. Fulfillment centers
+    3. Operational timezone
+5. Setup theme
+
+Check that new store is visible.
```

Comment:
what's visible where? in the list?

Reply:
fixed in this way
`Open your first store in a browser.`